### PR TITLE
Update kubevirt-api version

### PR DESCRIPTION
**Pull request by avivtur**
Signed-off-by: Aviv Turgeman <aturgema@redhat.com>
## 📝 Description

Updating the kubevirt-api to version 0.47
side effects: 
`MigrationPolicy` resource has changed in the `spec.selectors` section which caused some changed to be aligned with the newest version

## 🎥 Demo

> Please add a video or an image of the behavior/changes

### CODE REVIEW

File: src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/components/MigrationPolicyProjectLabels.tsx [hunk 1]
```diff
@@ -28,7 +28,11 @@ const MigrationPolicyVirtualMachineLabels: React.FC<MigrationPolicyVirtualMachin
   const { t } = useKubevirtTranslation();
   const { createModal } = useModal();
 
-  const mpNamespaceMatchLabels = mp?.spec?.selectors?.namespaceSelector?.matchLabels;
+  const mpNamespaceSelector = mp?.spec?.selectors?.namespaceSelector;
+  console.log(
```
Comment (upalatucci) [inline src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/components/MigrationPolicyProjectLabels.tsx:32]: Do we need this console log?

Comment (avivtur) [reply src/views/migrationpolicies/details/tabs/details/components/MigrationPolicyDetailsSection/components/MigrationPolicyProjectLabels.tsx]: removed, good catch buddy :)

**Review by upalatucci — APPROVED** [review]
Great man

**Pull request merged**